src/service-module/getters.js

```diff
@@ -1,4 +1,4 @@
-import getFilter from 'feathers-query-filters'
+import { filterQuery } from '@feathersjs/commons'
 import sift from 'sift'
 import { sorter, select, _ } from 'feathers-commons'
```

Comment:
I believe this is also `@feathersjs/commons` It could probably be merged with the above import on L1